presto-docs/src/main/sphinx/presto_cpp/properties.rst

```diff
@@ -323,3 +323,37 @@ This only applies if ``system-mem-pushback-enabled`` is ``true``.
 
 Specifies the amount of memory to shrink when the memory pushback is
 triggered. This only applies if ``system-mem-pushback-enabled`` is ``true``.
+
+Environment Variables As Values For Worker Properties
+-----------------------------------------------------
+
+This section applies to worker configurations in the config.properies file
+and catalog property files only.
+
+The value in a key-value par may reference an environment variable by using
+a leading `$` followed by enclosing the environment variable name in brackets (`{}`).
+
+```
```

Comment:
Do you want the ` and space before and after in the code block? This is how the block looks in a local doc build. If this is what you want then please resolve this comment, if not adjust the formatting. 
<img width="776" alt="Screenshot 2024-12-02 at 10 14 10 AM" src="https://github.com/user-attachments/assets/11309647-43b6-4fc0-946c-ab974c76910e">